invenio_stats/config.py

```diff
@@ -35,11 +37,18 @@
 )
 """Default exchange for message queue."""
 
-STATS_INDICIES_PREFIX = 'events'
+STATS_INDICES_PREFIX = 'events'
 """Allowed event types."""
 
 STATS_REGISTER_RECEIVERS = True
 """Register signal receivers."""
 
 STATS_INDICES_SUFFIX = '%Y.%W',
 """Suffix of indices."""
+
+CELERY_BEAT_SCHEDULE = {
+    'indexer': {
+        'task': 'invenio_stats.tasks.index_events',
+        'schedule': timedelta(seconds=5),
```

Comment:
The celery beat doesn't take into account if the task was finished or not, i.e. if the task didn't finish with-in 5s then you risk having tasks dog piling.